### DISCUSSION TITLE
显示视频投稿时间位置优化

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#4160] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/14146754/234205156-eda7ad13-a564-423b-993a-34ee43bb6e63.png)
遇到up主名字比较长的时候时间会显示不全
可以把时间单独挪到下面一行

这样的话显示比较全,
投稿时间的长度和播放量的长度差不多,卡片之间看起来也不会很挤

----
刚才看了下,这功能原来是替换up名来实现的
这样的话能否改成替换播放量那行来实现呢?然后文本替换功能支持换行符

**Comment by the1812** [comment]
@wisokey

**Comment by wisokey** [comment]
在pr #4136 里提交了支持自定义替换字符串的格式，而且这个提交好像已经在预览版本发布了。不过我并没有添加换行符的解析。你可以先尝试自定义格式来缩短下字符串长度。我应该不久会加上一些转义字符的支持。

**Comment by wisokey** [comment]
对于播放量那一行不是一个完整的字符串，这个没有办法去做替换。如果要加在那一行需要去改动UI。而对于UI的改动这方面我目前还没有这个打算。

**Comment by wisokey** [comment]
@the1812 这个提议能关掉吗。因为我发现即使我使用了换行符也不能实现换行效果。

**Reply by the1812** [reply]
换行要改 CSS white-space 的，有试过吗

**Reply by wisokey** [reply]
旧版：
![image](https://user-images.githubusercontent.com/57711857/234864519-f1d94d5b-4704-41b3-bebc-3cfa45db02c6.png) 
新版：
![image](https://user-images.githubusercontent.com/57711857/234886971-8cf40187-1770-4b11-b733-7f3b610b8dfb.png)
这个看起来很糟糕
